### PR TITLE
made home page

### PR DESCRIPTION
**Pull request by ninawang4**
Home page has
- upcoming meetings
- incoming requests
- user statistics
- a button leading to search (pls make sure this works)

### CODE REVIEW

File: client/components/Home/HomePage.vue [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    <section class="row">
+        <div class="column">
+            <header>
+            <h1>Welcome to Alumni Connector!</h1>
+            </header>
+        </div>
+        <div class="column"> 
+            <button
+                v-if="$store.state.userId"
+                onclick="window.location.href=`${window.location.href}search`"
```
Comment (tejalreddy) [inline client/components/Home/HomePage.vue:15]: I think there is another way to make a router link than this. See the usercard component
 ```
<router-link 
        :to="'/profile/'+ user._id"
 >
```

File: client/components/Meeting/MeetingPage.vue [hunk 2]
```diff
@@ -109,10 +109,10 @@
             that.pastMeetings = Object.values(result);
         });
 
-        const paramsPastNoFeedback = {method: 'GET', url: '/api/timeblock/met/check'}
-        this.request(paramsPastNoFeedback).then(function(result) {
-            that.meetingsWithoutFeedback = Object.values(result);
-        });
+        // const paramsPastNoFeedback = {method: 'GET', url: '/api/timeblock/met/check'}
```
Comment (tejalreddy) [inline client/components/Meeting/MeetingPage.vue:112]: Delete instead of commenting it out

File: client/components/Home/HomePage.vue [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    }
+  };
+  </script>
+
+  <style scoped>
+
+section {
+  display: flex;
+  flex-direction: column;
+  /* background-color: #abd6be; */
```
Comment (tejalreddy) [inline client/components/Home/HomePage.vue:156]: Delete instead of commenting out

**Review by tejalreddy — APPROVED** [review]
Good besides a few nits

File: client/components/Home/HomePage.vue (hunk 1 above)
Comment (ninawang4) [reply client/components/Home/HomePage.vue:15]: when i did router link it got rid of my styling lol :(

**Pull request merged**